app/scripts/directives/datepicker.js

```diff
@@ -16,7 +16,7 @@ angular
         $scope.updateTimeStamp = function (timestamp) {
           //For Graduation date question, set the day to 1. The API needs the day but that could change in the future.
           timestamp = $scope.monthYearOnly
-            ? moment(new Date(timestamp)).set('date', 1)
+            ? moment(new Date(timestamp)).set('date', 10)
```

Comment:
If this is changing, the comment above should also change.

Reply:
Ah yes good catch